source/layouts/blog.erb

```diff
@@ -35,7 +35,7 @@
     </div>
   </article>
 
-  <% if current_article.date > Date.new(2019,1,1) %>
+  <% if current_article.date > Date.new(2019,0,1) %>
```

Comment:
I guess this should be a 1 because I think Ruby is `1 indexed` for dates? https://ruby-doc.org/stdlib-2.1.9/libdoc/date/rdoc/Date.html#method-c-new

I know nothing about ruby 🙈

Reply:
Oh right, thank you! I was inferring that it's the same in JavaScript - good catch ✨

Reply:
That's what I thought first too, I was like "Good catch, JS it should be a 0" 😂